lib/groups/canPay.js

```diff
@@ -0,0 +1,50 @@
+// Includes
+const http = require('../util/http.js').func
+
+// Args
+exports.required = ['group', 'member']
+exports.optional = []
+
+// Docs
+/**
+ * 🔓 Get if a user can be paid from group funds.
+ * @category Group
+ * @alias canPay
+ * @param {number} group - The id of the group.
+ * @param {number} member - The member to check payout status for.
+ * @returns {Promise<PayoutAllowedList>}
+ * @example const noblox = require("noblox.js")
+ * const groupShout = await noblox.getShout(1)
+ **/
+
+// Define
+function getShout (group, member, jar) {
```

Comment:
Apart from the fact that this was forgotten, I'm not sure if "canPay" is a good name for this method.

Maybe `getGroupPayoutEligibility`?

Reply:
Yeah, that makes more sense